docs/workflow.rst

```diff
@@ -3,255 +3,182 @@
 Standard Workflow
 #################
 
-The workflow is setup to run the fitting on many sources efficiently by
+The workflow is set up to run the fitting on many sources efficiently by
 splitting the full catalog into a number of smaller files.  This allows
 distributing the fitting across cores.  There are manual steps to allow
 for the refitting, fixing issues, etc without rerunning everything.  This
 workflow has been tested on large (e.g., PHAT) and small (e.g. METAL)
 datasets.
 
-****************************
-Production Conda Environment
-****************************
 
-Using a dedicated conda environment for production BEAST runs may be
-desirable.  Such an environment provides a way to ensure that
-production runs are reproducible by fixing the versions of all the
-software used.  The instructions below assume that the `astroconda channel
-<https://astroconda.readthedocs.io/>`_ is being used.
-
-Create a conda environment.  Here we name it to include the BEAST version.
-
-  .. code-block:: console
-
-    $ conda create -n beast_v1.3 python=3.6
-
-Activate the environment after all the packages are finished installing.
-
-  .. code-block:: console
+*****
+Setup
+*****
 
-    $ source activate beast_v1.3
+Setup a working location. For reference, there are examples in beast/examples.
 
-Install dependencies using conda (better for speed)
+In this location, you will need a datamodel.py parameter file.
+These parameters are described in the beast :ref:`setup documentation
+<beast_setup_datamodel>`.
 
-  .. code-block:: console
 
-    $ conda install astropy scipy h5py matplotlib cython
 
-Next, install the BEAST.  You have three options:
+**********************************
+Source Density or Background Level
+**********************************
 
-Option 1: Use pip to install the production version of the beast (currently v1.3)
+The data need to have source density information added as it is common
+for the observation model (scatter and bias) to be strongly dependent
+on source density due to crowding/confusion noise.  The background may
+also be important in some situations, so there is code to calculate it as well.
 
-  .. code-block:: console
+Adding source density to observations
+=====================================
 
-    $ pip install beast==1.3
+Create a new version of the observations that includes a column with the
+source density.  The user chooses one band to use as the reference, and chooses
+the magnitude range of sources to use for calculating the source density
+(generally, this would be the range over which the catalog is complete).  The
+user can also choose a band for which sources that have 'BAND_FLAG == 99' are
```

Comment:
What does BAND_FLAG=99 mean?

Reply:
changed it to `[band]_FLAG` to hopefully clarify that it's a column in the photometry data table.